kernel/packages/scene-system/stateful-scene/SceneStateDefinitionSerializer.ts

```diff
@@ -1,63 +1,28 @@
-import { CLASS_ID } from "decentraland-ecs/src";
-import { SceneStateDefinition } from "./SceneStateDefinition";
-import { Component, ComponentId, EntityId } from "./types";
-
-export class SceneStateDefinitionSerializer {
-
-  static toStorableFormat(state: SceneStateDefinition): any {
-    const entities = []
-    for (const [entityId, entityComponents] of state.getState().entries()) {
-      const components = []
-      for (const [componentId, componentData] of entityComponents.entries()) {
-        components.push({ type: idToHumanReadableType(componentId), value: componentData })
-      }
-      entities.push({ id: entityId, components })
-    }
-    return { entities }
-  }
-
-  static fromStorableFormat(data: any): SceneStateDefinition {
-    const sceneState = new SceneStateDefinition()
-    for (const entity of data.entities) {
-      const id: EntityId = entity.id
-      const components: Component[] | undefined = entity.components
-        ?.map((component: any) => ({
-          componentId: humanReadableTypeToId(component.type),
-          data: component.value,
-        }))
-      sceneState.addEntity(id, components)
+import { SerializedSceneState } from 'shared/apis/SceneStateStorageController/types'
+import { SceneStateDefinition } from './SceneStateDefinition'
+import { Component, EntityId } from './types'
+
+export function serializeSceneState(state: SceneStateDefinition): SerializedSceneState {
```

Comment:
I did 2 things here:
1. `SerializedSceneState` is now being used to send data from the worker to kernel. It will later be transformed again in order to store it. The idea is to decouple storage format from transfer format
2. Embracing functions